.ci/Jenkinsfile

```diff
@@ -52,6 +52,8 @@ pipeline {
     string(name: 'HELM_VERSION', defaultValue: '3.5.2', description: 'SemVer version of Helm to be used.')
     string(name: 'HELM_KIND_VERSION', defaultValue: '0.10.0', description: 'SemVer version of Kind to be used.')
     string(name: 'HELM_KUBERNETES_VERSION', defaultValue: '1.18.2', description: 'SemVer version of Kubernetes to be used.')
+    string(name: 'KUBERNETES_AUTODISCOVER_KIND_VERSION', defaultValue: '0.10.0', description: 'SemVer version of Kind to be used.')
```

Comment:
wdyt about reusing existing Jenkins params? We could remove those two new lines, and in your script, use the values.

If you see it convenient, you could rename the existing Jenkins params & env vars (https://github.com/elastic/e2e-testing/pull/1064/commits/6f8cd442840d28acd5b942e720efa5bc71555ad7#diff-ef8b2f5e4d21d81c0eba4754dea3a3dfc2c67d26284f195165b650cf4aff7beaR79-R81) with a reusable namespaced name, something like `KIND_VERSION`. wdyt?

Reply:
Done in a64429347c8317b4b616f51f09ad51b7d6a53c67

Are `HELM_*` variables used somewhere else to invoke these jobs? Should I keep them for backwards compatibility?